components/Snackbar/src/private/MDCSnackbarMessageViewInternal.h

```diff
@@ -53,7 +53,19 @@ typedef void (^MDCSnackbarMessageDismissHandler)(BOOL userInitiated,
 /**
  If the snackbar view should be anchored to the bottom of the screen. Default is YES.
  */
-@property(nonatomic) BOOL anchoredToScreenEdge;
+@property(nonatomic) BOOL anchoredToScreenBottom;
+
+/**
+ The layout constraint which determines how far the snackbar is from the leading edge of the screen.
```

Comment:
Nit: %s/snackbar/Snackbar/g

(apply change throughout)

Reply:
Agreed, sorry for the confusion (I wrote it poorly).

Reply:
I already did it for a lot of occurrences... I can push it up and you can let me know if you want me to re-lowercase them...

Reply:
Oh, it's fine. I meant that you didn't need to do unnecessary work. 🤷‍♀️